app/liquid/liquid_renderer.rb

```diff
@@ -11,7 +11,9 @@ def initialize(page, layout: nil, request_country: nil, member: nil, url_params:
   end
 
   def render
-    template.render( data ).html_safe
+    Rails.cache.fetch(cache_key) do
+      template.render( data ).html_safe
+    end
```

Comment:
Hi @NealJMD this single cache block saves us a lot of processing time. The `cache_key` is built from two records: `page` and `liquid_layout`. If either record is `touch`ed, then the cache invalidates.